projects/packages/paypal-payments/package.json

```
@@ -21,5 +21,19 @@
 		"build-production-js": "echo 'Not implemented.'",
 		"clean": "true"
 	},
-	"devDependencies": {}
+	"dependencies": {
+		"@automattic/jetpack-shared-extension-utils": "workspace:*",
```

Should we add a dependency to , since that's needed for the build?

```suggestion
		"@automattic/jetpack-shared-extension-utils": "workspace:*",
		"@automattic/jetpack-webpack-config": "workspace:*",
```

---

Thanks for flagging this. I've added it in [515815e](https://github.com/Automattic/jetpack/pull/43413/commits/515815e86ee65b6388567ec3e481062a85966c84).